config/crds/kots.io_licenses.yaml

```diff
@@ -70,6 +85,8 @@ spec:
                 type: boolean
               isDisasterRecoverySupported:
                 type: boolean
+              isEmbeddedClusterDownloadEnabled:
+                type: boolean
```

Comment:
Just noticed this, it was the most recent addition - guessing we just missed regen? I can ommit and/or send along a separate pr for this if folks prefer.